app/src/interfaces/translations/translation-form.vue

```diff
@@ -0,0 +1,186 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { isEmpty } from 'lodash';
+import { usePermissions } from '@/composables/use-permissions';
+import { type DisplayItem } from '@/composables/use-relation-multiple';
+import { type RelationM2M } from '@/composables/use-relation-m2m';
+import LanguageSelect from './language-select.vue';
+
+const {
+	languageOptions,
+	disabled,
+	relationInfo,
+	getItemWithLang,
+	displayItems,
+	fetchedItems,
+	getItemEdits,
+	isLocalItem,
+	remove,
+	updateValue,
+} = defineProps<{
+	languageOptions: Record<string, any>[];
```

Comment:
TIL: This is actually allowed now and does not loose reactivity. 👀 

https://vuejs.org/guide/components/props#reactive-props-destructure